### ISSUE TITLE
poki.com: ads

### ISSUE THREAD

**Issue by Excrubulent** [uBlockOrigin/uAssets#13603]
### Prerequisites

- [X] I read and understood the [policy as to what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets)
- [X] I verified that the issue has not already been reported (use [this button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out)
- [X] I forced an update of my filter lists (**how to do this:** [click _"Purge all caches"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches), then [click _"Update now"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now))
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I verified that the issue is not caused by removing any of the default lists
- [X] I did not enable external filter lists, or I verified that the issue still occurs without enabling external filter lists
- [X] I do not have custom filters/rules, or I verified that the issue still occurs without custom filters/rules
- [X] I am not using [another content blocker along uBO](https://twitter.com/gorhill/status/1033706103782170625) (also mind browser built-in blockers)
- [ ] I did not answer truthfully to **all** the above checkpoints

### URL address of the web page

`https://poki.com/en/g/ducklife-space`

### Category

ads

### Describe the issue

Ads play periodically when playing this game.  Purging and updating the lists seemed to change the behaviour slightly, as now some ads are blacked out but still play audio.

Fastest way to reproduce the issue is to simply refresh the page and start the game again, I've found.

Some ads on the website flicker black & disappear, but some still play entirely.  Some games seem to have all their ads blocked, but this one seems to get most of them through, although that may simply be due to the small sample I took.

I'm also using noscript.

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![image](https://user-images.githubusercontent.com/11778078/172283117-707f8261-4ceb-40e3-9e22-1c8d1f71fec8.png)
![image](https://user-images.githubusercontent.com/11778078/172283136-8d405177-f69c-44bb-9551-4e1ecc809564.png)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.42.4
Firefox: 101
filterset (summary): 
  network: 83458
  cosmetic: 43773
  scriptlet: 17137
  html: 676
listset (total-discarded, last updated): 
  default: 
    user-filters: 0-0, never
    ublock-filters: 32342-42, 22h.9m
    ublock-badware: 4170-3, 22h.8m
    ublock-privacy: 228-0, 7d.1h.47m
    ublock-abuse: 75-0, 7d.1h.46m
    ublock-unbreak: 1807-0, 22h.7m
    easylist: 67948-584, 22h.11m
    easyprivacy: 27152-579, 22h.10m
    urlhaus-1: 8823-0, 22h.6m
    plowe-0: 3671-3, 11d.20h.30m
    ublock-quick-fixes: 203-0, 22h.5m
filterset (user): [empty]
modifiedUserSettings: 
  advancedUserEnabled: true
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 558 ms (selfie)
  maxAssetCacheWait: 160 ms
popupPanel: 
  blocked: 246
  blockedDetails: 
    poki.com: 3
    2mdn.net: 17
    33across.com: 6
    3lift.com: 6
    4dex.io: 2
    adnxs.com: 6
    amazon-adsystem.com: 4
    casalemedia.com: 4
    doubleclick.net: 47
    google.com: 9
    googlesyndication.com: 22
    gstatic.com: 23
    gvt1.com: 2
    imasdk.googleapis.com: 2
    indexww.com: 2
    onetag-sys.com: 6
    openx.net: 6
    poki.io: 47
    pubmatic.com: 6
    richaudience.com: 6
    rubiconproject.com: 6
    sharethrough.com: 4
    spotxchange.com: 4
    unity3d.com: 4
    youtube.com: 2
```
</details>

**Comment by mapx-** [comment]
I refreshed 3 times, no ad on my side.
Keep enabled only uBO (default settings/ filter lists), disable noscript, test again

**Comment by Excrubulent** [comment]
Just tried it again, disabled noscript for the entire tab and got the same result.

Also I realise my repro instructions weren't complete - for me the ad doesn't play until I click PLAY in the game.  I'm not sure if you need to play the game first to get it to happen, because it remembers your save data.  It also happens after playing a couple of levels.

**Comment by Excrubulent** [comment]
Yeah, just tried it in a private tab to clear the game data - my kid plays this so I couldn't just delete the save data.

It won't play ads the first time you open it.  You need to play the game for a little bit, then on one of your restarts the ads will appear.

Thereafter it appears to give you ads whenever refreshing the page, presumably to prevent you from getting around the ads that way.

**Comment by mapx-** [comment]
playing some minutes, training, etc, refreshing => still no ad

**Comment by Excrubulent** [comment]
Okay, thanks for trying it.  Sorry, I don't know why it's different.  I can make a video of it if that would help.

**Comment by Mahadlo1** [comment]
can you make it for venge?
